src/Facades/PageFactory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Astrotomic\Stancy\Facades;
+
+use Illuminate\Support\Facades\Facade;
+use Astrotomic\Stancy\Contracts\PageFactory as AstrotomicPageFactory;
+
+/**
+ * @method PageContract make(array $data = [], ?string $page = null) Creates a Page
+ * @method PageContract makeFromSheet(string $collection, string $name, ?string $page = null) Creates a Page from a Sheet
+ * @see \Astrotomic\Stancy\Contracts\PageFactory
```

Comment:
```suggestion
 *
 * @see \Astrotomic\Stancy\Factories\PageFactory
```

Reply:
@Gummibeer I used alias for `Astrotomic\Stancy\Contracts\PageFactory` like other facades.